### ISSUE TITLE
Local video does not update isVideoEnabled when I get lost access to the current camera.

### ISSUE THREAD

**Issue by allys00** [aws/amazon-chime-sdk-component-library-react#873]
### What happened and what did you expect to happen?

**What happens:**
When for some reason I disconnect my current camera, I get an alert "_**Video input device which was active is no longer available, stopping video**_", after that I select another camera and nothing happens, I can only get the video again when I use `toggleVideo `twice.
**What could happen:** 
When the current camera is disconnected, it could just switch to the next available camera, but if for security reasons this is not possible, if it manually selects another camera it should show video, or at least update the `isVideoEnabled `to false.

How do I solve this in my code when I select a change device and put this code` audioVideo?.startLocalVideoTile();`
next to the `audioVideo?.startVideoInput(_cameraId);`

### Have you reviewed our existing documentation?

- [X] Amazon Chime SDK for JavaScript GitHub [issues](https://github.com/aws/amazon-chime-sdk-js/issues)
- [X] Amazon Chime SDK React Components Library GitHub [issues](https://github.com/aws/amazon-chime-sdk-component-library-react/issues)
- [X] Storybook [documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/introduction--page)
- [X] README [page](https://github.com/aws/amazon-chime-sdk-component-library-react#readme)
- [X] How-to [documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/how-tos--page)
- [X] React SDK meeting [demo](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/meeting#readme---react-meeting-demo)

### Reproduction steps

Enter a meeting and remove your current camera.

![image](https://user-images.githubusercontent.com/29717970/224502178-4d78e0ee-597a-4488-97ff-0234ec0cb455.png)
![image](https://user-images.githubusercontent.com/29717970/224502219-431e7111-04bf-485c-8c60-e0cbded125cb.png)
![image](https://user-images.githubusercontent.com/29717970/224502247-c972b969-9722-4f94-b485-7523eefff66b.png)

### Amazon Chime SDK React Components Library version

3.5.0 or 3.6.0

### What browsers are you seeing the problem on?

Chrome

### Browser version

Versão 110.0.5481.180 

### Device Information

Notebook Windows

### Meeting and Attendee ID Information.

 ```
const JoinInfo = {
        Attendee: {
          AttendeeId: "be32e57b-7c05-ad9e-039c-6161a511925e",
          ExternalUserId: "df7742dc-051d-45fd-9fa7-f2dbe7613a67",
          JoinToken:
            "YmUzMmU1N2ItN2MwNS1hZDllLTAzOWMtNjE2MWE1MTE5MjVlOjJkMGYyNDBiLWZhNzQtNGZhYy05NTY5LTEzZDQyNDc2NTdiYw",
        },
        Meeting: {
          ExternalMeetingId: "efade8a8-fe02-4fd0-9115-f0315d1f5c29",
          MediaPlacement: {
            AudioFallbackUrl:
              "wss://haxrp.m2.ue1.app.chime.aws:443/calls/d40539a9-82c5-4b30-8208-8d4220fe0706",
            AudioHostUrl:
              "514841c152b39c4f40a936f6a2238456.k.m2.ue1.app.chime.aws:3478",
            EventIngestionUrl:
              "https://data.svc.ue1.ingest.chime.aws/v1/client-events",
            ScreenDataUrl:
              "wss://bitpw.m2.ue1.app.chime.aws:443/v2/screen/d40539a9-82c5-4b30-8208-8d4220fe0706",
            ScreenSharingUrl:
              "wss://bitpw.m2.ue1.app.chime.aws:443/v2/screen/d40539a9-82c5-4b30-8208-8d4220fe0706",
            ScreenViewingUrl:
              "wss://bitpw.m2.ue1.app.chime.aws:443/ws/connect?passcode=null&viewer_uuid=null&X-BitHub-Call-Id=d40539a9-82c5-4b30-8208-8d4220fe0706",
            SignalingUrl:
              "wss://signal.m2.ue1.app.chime.aws/control/d40539a9-82c5-4b30-8208-8d4220fe0706",
            TurnControlUrl: "https://ccp.cp.ue1.app.chime.aws/v2/turn_sessions",
          },
          MediaRegion: "us-east-1",
          MeetingId: "d40539a9-82c5-4b30-8208-8d4220fe0706",
        },
      };
```

### Browser console logs

![image](https://user-images.githubusercontent.com/29717970/224502363-70b64864-9757-4013-a2b5-56d3935f907a.png)
[192.168.1.41-1678554941912.log](https://github.com/aws/amazon-chime-sdk-component-library-react/files/10949041/192.168.1.41-1678554941912.log)

### Add any other context about the problem here.

If you confirm that it is a problem, I would like to contribute with the solution if you need ^^

**Comment by michhyun1** [comment]
Hey @allys00 thanks for bringing this to our attention. This seems to be a bug on our end - it seems like the VideoInputProvider did not implement the `VideoInputDidChange` observer callback from the JS SDK. 

That's probably why the VideoInputProvider is not updated when the device is changed on the JS SDK - when you disconnect the video device. 

You are correct in assuming that we cannot just switch to the next video device, as that is a major privacy concern. So that is expected. However, the value of isVideoEnabled should be updated accordingly when the video input device is updated. I'll mark this as a bug and we'll try to address this bug in the next release. Thanks.

Feel free to submit a pull request if you'd like, otherwise I'll backlog this item for the next release.

**Comment by michhyun1** [comment]
Actually @allys00 I realized that you are calling the audioVideo `startVideoInput` call directly. I would highly recommend using the meeting manager to make those calls to start and stop video instead of directly calling the audioVideo object. 

Instead of doing `audioVideo?.startVideoInput(_cameraId);` , I'd recommend doing `meetingManager.startVideoInputDevice(_cameraId)`.

**Comment by michhyun1** [comment]
Can you try checking out the main branch and seeing if tis fixed your issue. Thanks!

**Comment by xuesichao** [comment]
PR #878  is merged and is published in 3.6.0 release. Closing this now.
